Adyen/Utilities/Localization.swift

```diff
@@ -40,6 +40,30 @@ public func ADYLocalizedString(_ key: String, _ parameters: LocalizationParamete
     return String(format: result, arguments: arguments)
 }
 
+/// Returns a localized string for the given key, and optionally uses it as a template
+/// in which the remaining argument values are substituted.
+/// This method will try first to get the string from main bundle.
+/// If no localization is available on main bundle, it'll return from internal one.
+///
+/// :nodoc:
+///
+/// - Parameters:
+///   - key: The key used to identify the localized string.
+///   - parameters: The localization parameters.
+///   - arguments: The arguments to substitute in the templated localized string.
+/// - Returns: The localized string for the given key, or the key itself if the localized string could not be found.
+public func ADYLocalizedString(_ key: ADYLocalizationKey, _ parameters: LocalizationParameters?, _ arguments: CVarArg...) -> String {
+    let possibleInputs = buildPossibleInputs(key.key, parameters)
```

Comment:
You can just reuse `public func ADYLocalizedString(_ key: String, _ parameters: LocalizationParameters?, _ arguments: CVarArg...) -> String` right?

Reply:
Yes, but I did not want to make changes in the whole project before we agree on the script :)